### PR TITLE
Import functionality from govuk guix

### PR DESCRIPTION
**Pull request by cbaines**
Copy over the govuk and govuk-connect scripts from the govuk-guix repository. The intention is to keep the functionality roughly the same, and then remove the original versions from the govuk-guix repository.

This follows on from some discussion here: https://github.com/alphagov/govuk-guix/issues/36

### CODE REVIEW

File: bin/govuk [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+PATH="$GOVUK_EXEC_PATH:$PATH"
+
+COMMAND="$1"
+
+log "COMMAND is $COMMAND"
+
+if [ "$COMMAND" = "" ]; then
+    echo "Please specify a command, e.g.
+
```
Comment (issyl0) [inline bin/govuk]: I think the indentation went a bit wrong here?

File: bin/govuk [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+if [ "$COMMAND_FULL_PATH" = "" ]; then
+    echo "Command $COMMAND not found"
+    exit 1
+fi
+
+log "COMMAND_FULL_PATH is $COMMAND_FULL_PATH"
+
+# Determine if guile is available
+GUILE=""
```
Comment (issyl0) [inline bin/govuk]: For the uninitiated, what does guile do?

Comment (cbaines) [reply bin/govuk]: This relates to the `govuk-guix` bits. It's used if you run `govuk data` and have Guix available.

File: bin/govuk (hunk 1 above)
Comment (cbaines) [reply bin/govuk]: In what way? There's some whitespace, but that's intended to make the output like:
```
→ ./bin/govuk 
Please specify a command, e.g.

  govuk connect

```

Comment (issyl0) [reply bin/govuk]: Why not just:

```suggestion
Please specify a command, eg `govuk connect`
```

Comment (cbaines) [reply bin/govuk]: That's fine with me.

**Pull request merged**